interface-definitions/service_ipoe-server.xml.in

```diff
@@ -189,6 +189,14 @@
           #include <include/accel-ppp/snmp.xml.i>
           #include <include/generic-description.xml.i>
           #include <include/name-server-ipv4-ipv6.xml.i>
+          <node name="log">
+            <properties>
+              <help>Logging IPoE Server</help>
```

Comment:
This isn't a good grammar. I'd say "IPoE server logging". This applies to all instances of that string.